.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+#These are the default settings for stalebot. Please update
+#comments if this file is modified. https://github.com/apps/stale
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
We can try this but I wonder if this will close tickets while people are on vacation.  Wondering if 14 days would help avoid that.  Thoughts?

Reply:
I'm open to make it longer, but this already has a long stale period of 60 days of activity so really you are talking over two months of inactivity. Can also re-open if needed.

Reply:
@pklinef with the amount of approvers I'm recommending we keep it as it is and adjust later if necessary. I think we can have people give it activity if somebody is on an extended vacation, but hopefully people consider this before vacation now :-)

Reply:
I think the default will work well for PRs.  Stale issues I could see being closed but we can always reopen them when we get back.